Hi3Helper.Core/Lang/en_US.json

```diff
@@ -74,6 +74,8 @@
     "UnhandledSubtitle2": "Oops, it seems like you've been disconnected from the Internet~ Or is it something else?",
     "UnhandledTitle3": "Game Crashed",
     "UnhandledSubtitle3": "The game has crashed with error details below:",
+    "UnhandledTitle4": "Warning",
+    "UnhandledSubtitle4": "This isn't a major issue, but we thought we should let you know:\r\n\r\nDue to miHoYo A/B testing, Collapse does not support reading the following key: App_Settings_h2319593470.\r\n\r\nWe apologize for the inconvenience and thank you for your understanding.",

```

Comment:
There seems to be too much spacing between paragraphs, especially the last one. A single `\r\n` should do just fine.

Reply:
See above